docs/en/observability/inspect-metric-anomalies.asciidoc

```diff
@@ -37,25 +35,40 @@ By default, the Kubernetes partition field `kubernetes.namespace` is selected.
 +
 Partitions allow you to create independent models for different groups of data 
 that share similar behavior. For example, you may want to build separate models 
-for machine type, or cloud availability zone, so that anomalies are not weighted 
+for machine type or cloud availability zone so that anomalies are not weighted 
 equally across groups.
 +
-5. Click *Enable Jobs*.
-6. You're now ready to explore your metric anomalies. Click *View anomalies* to 
-view the {ml-docs}/ml-gs-results.html[Anomaly Explorer in Machine Learning].
-
+5. By default, {ml} jobs analyze all of your metric data. You can filter to
+analyze specific metrics. 
```

Comment:
This step just quotes the UI. Can we add an example here of what data users may want to filter on, and why? Something similar to what you did in step 4. I have no idea what that may be, but hopefully someone else tagged in this PR does 😄

Reply:
Good point. @sorantis I added a little more context to the step. Could you give it a review? Thanks.